book/modules/module12.tex

```diff
@@ -21,7 +21,7 @@
 is invertible if there exists an inverse function that when composed with the original function
 produces the identity function and vice versa.
 
-\SavedDefinitionRender{InverseFunction}
+\SavedDefinitionRender{InverseFunction}\index[definitions]{Linear transformation!inverse}
```

Comment:
These are things about functions, don't restrict them to linear transformations.

Reply:
normal entry no sub